Jenkinsfile.promote

```diff
@@ -151,7 +151,7 @@ pipeline {
         //                 githubscm.createBranch(getSnapshotBranch())
                         
         //                 // Update version
-        //                 sh "./hack/bump-version.sh ${nextVersion}"
+        //                 sh "./hack/bump-version.sh ${nextVersion} true ${getBuildBranch() == 'master'}"
```

Comment:
Just out of curiosity, but we want to keep commented code?

Reply:
[The whole stage](https://github.com/kiegroup/kogito-cloud-operator/blob/1c815e7f8dc56ea3053933bf9d0621568dc253f2/Jenkinsfile.promote#L138-L170) is commented. It's prefaced with:
> // That part is commented as it messes with PRs on operatorhub which may require some changes
> // Commented for now